obspy/taup/velocity_model.py

```diff
@@ -484,6 +483,128 @@ def readTVelFile(cls, filename):
                              cls.default_cmb, cls.default_iocb, 0,
                              maxRadius, True, layers)
 
+    @classmethod
+    def read_nd_file(cls, filename):
+        """
+        This method reads in a velocity model from a "nd" ASCII text file, the
+        format used by Xgbm. The name of the model file for model "modelname"
+        should be "modelname.nd".
```

Comment:
Can you please reformat the docstring to contain a one line summary, e.g.:

```python
def read...():
    """
    Method to read in a velocity model from a "nd" ASCII text file.

    This method....
"""

Then there will be a proper one line description in the documentation for this function. See here for details: https://www.python.org/dev/peps/pep-0257/

Also best but a blank line where you want a line to end (e.g. two lines down) - sphinx otherwise will mash the lines together has it cannot determine where the line break should be.